docs/manual/comparison_with_pandas.rst

```diff
@@ -118,19 +118,19 @@ In ``pandas`` every frame has a row index, and if a filtration is executed, the
     # pandas
     df.loc[df['A']>3]
```

Comment:
What about adding ">>>" here?

Reply:
I've added `>>>` only where Sphinx was complaining about not being able to lex the snippet. There are outstanding issues #2625 and #2626 to improve rendering of these code fragments and the style comparison pages in general.